### PR TITLE
ARUHA-1081: moved log levels specification for specific packages to l…

### PR DESCRIPTION
**Pull request by v-stepanov**
…og4j.properties;

[ARUHA-1081: DEBUG logging in Nakadi doesn't work](https://techjira.zalando.net/browse/ARUHA-1081)

## Description
Fixed DEBUG logging. After we switched to newer version of spring-boot - our logging configs in application.yml stopped working. I moved that now to log4j.properies.

## Review
- [ ] Tests
- [ ] Documentation
- [x] CHANGELOG

## Deployment Notes
-

### CODE REVIEW

File: src/main/resources/log4j.properties [hunk 1]
```diff
@@ -18,5 +18,6 @@ log4j.category.org.hibernate.validator.internal.util.Version=WARN
 log4j.category.org.springframework.boot.actuate.autoconfigure.CrshAutoConfiguration=WARN
 log4j.category.org.springframework.boot.actuate.endpoint.jmx=WARN
 log4j.category.org.thymeleaf=WARN
-# Do not print consumer configuration on each consumer creation
-log4j.category.org.apache.kafka.clients.consumer.ConsumerConfig=WARN
\ No newline at end of file
+log4j.category.org.zalando.nakadi=DEBUG
+log4j.category.org.zalando.nakadi.config=INFO
+log4j.category.org.apache.kafka=WARN
```
Comment (adyach) [inline src/main/resources/log4j.properties]: \n

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -8,10 +8,6 @@ management:
 logging:
```
Comment (adyach) [inline src/main/resources/application.yml]: do we really need this?

Comment (v-stepanov) [inline src/main/resources/application.yml]: no
https://github.com/zalando/nakadi/pull/735/commits/a8952f590155ecabaa2f136b1154ea8f5fcf3f28

File: src/main/resources/log4j.properties (hunk 1 above)
Comment (v-stepanov) [inline src/main/resources/log4j.properties]: 🤓 
https://github.com/zalando/nakadi/pull/735/commits/a8952f590155ecabaa2f136b1154ea8f5fcf3f28

**Comment by adyach** [comment]
👍

File: src/main/resources/log4j.properties (hunk 1 above)
Comment (v-stepanov) [inline src/main/resources/log4j.properties]: @adyach because we have WARN level for root kafka package anyway "org.apache.kafka"

**Comment by v-stepanov** [comment]
👍

**Comment by v-stepanov** [comment]
👍

**Comment by adyach** [comment]
:+1:

**Pull request merged**